LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 DataDrivenEcologicalSynthesis
+Copyright (c) 2020 Emily Kroft, Étienne Lacroix-Carignan, Jenna LeBlanc, Amanda Liczner & Indigo Tran
```

Comment:
Are we doing alphabetically with our last name? Also Oxford comma.

Reply:
Yeah, I thought alphabetically was the simplest way, and it was suggested by Timothé. We can create a poll on the use of the Oxford comma, that would be fun :P

Reply:
I'm for Oxford comma 🤚